bundles/org.openhab.binding.neohub/src/main/java/org/openhab/binding/neohub/internal/NeoHubHandler.java

```diff
@@ -209,15 +248,20 @@ protected NeoHubInfoResponse fromNeoHubFetchPollingResponse() {
      */
     private synchronized void lazyPollingSchedulerExecute() {
         fromNeoHubFetchPollingResponse();
+        
+        NeoHubInfoResponse myInfoResponse; 
+        synchronized (this) {
+            myInfoResponse = cachedInfoResponse;
+        }
```

Comment:
Synchronization isn't provided the thread safety here, instead it is the fact you are assigning cachedInfoResponse to a local variable that is giving you the thread safety. So please remove the synchronized block.

Reply:
eliminated cachedInfoResponse